pkg/apis/pxc/v1/pxc_types.go

```diff
@@ -58,6 +58,7 @@ const (
 type PXCScheduledBackup struct {
 	Image              string                        `json:"image,omitempty"`
 	ImagePullSecrets   []corev1.LocalObjectReference `json:"imagePullSecrets,omitempty"`
+	ImagePullPolicy    corev1.PullPolicy             `json:"imagePullPolicy,omitempty"`
```

Comment:
1. please add default value into CheckNSetDefaults
2. please use this value in all `pkg/pxc/backup/*.go` files

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/622/commits/59bd73bc66449d8e1aad668f7d654afef22c601b